### PR TITLE
added: juhair.json

### PR DESCRIPTION
**Pull request by juhair-cupcake**
It's a **Netlify** hosted site.
### Link to Website
https://juhair-dev.netlify.app/

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

**Comment by juhair-cupcake** [comment]
Is it accepted? @WilliamDavidHarrison

**Comment by WilliamDavidHarrison** [comment]
> Is it accepted? @WilliamDavidHarrison

Yes, please wait for a maintainer to merge it now.

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**
